tests/intern.ts

```diff
@@ -1,9 +1,14 @@
-export let loaderOptions = {
+export const loaders = {
+	'host-browser': 'node_modules/dojo-loader/dist/umd/loader.js',
```

Comment:
This isn't correct anymore since updating the dojo-loader packaging, it should be `node_modules/dojo-loader/loader.js`